### PR TITLE
feat(push): implement proper patchZone flag

### PR DESCRIPTION
**Pull request by hoebbelsB**
`patchZone` wasn't implemented correctly. fixed that :)

I did not add it as a third argument (` v$ | push: 'strategy'; renderCallback; patchZone`) since it's kind of useless in this scenario. We should think about having the push pipe taking an object as argument, or some kind of tuple type.
The current solution is a pain if you want to configure something in the view.

### CODE REVIEW

**Comment by hoebbelsB** [comment]
i know why the build is complaining. i make a separate pr for this

File: libs/template/src/lib/push/push.pipe.ts [hunk 1]
```diff
@@ -79,34 +83,13 @@ export class PushPipe<U> implements PipeTransform, OnDestroy {
```
Comment (Karnaukhov-kh) [inline libs/template/src/lib/push/push.pipe.ts:83]:
```suggestion
/** @internal */
```

File: libs/template/src/lib/push/push.pipe.ts [hunk 2]
```diff
@@ -79,34 +83,13 @@ export class PushPipe<U> implements PipeTransform, OnDestroy {
...
-              scope,
-              strategy: strategy.name,
-              patchZone: this.strategyProvider.config.patchZone ? null : false,
-            }
-          )
-        )
-      )
-      .subscribe();
+    this.subscription = this.handleChangeDetection();
   }
```
Comment (Karnaukhov-kh) [inline libs/template/src/lib/push/push.pipe.ts:94]:
```suggestion
/** @internal */
```

File: libs/template/src/lib/push/push.pipe.ts [hunk 3]
```diff
@@ -140,4 +123,38 @@ export class PushPipe<U> implements PipeTransform, OnDestroy {
   ngOnDestroy(): void {
     this.subscription.unsubscribe();
   }
+
```
Comment (Karnaukhov-kh) [inline libs/template/src/lib/push/push.pipe.ts:126]:
```suggestion
/** @internal */
```

**Review by Karnaukhov-kh — COMMENTED** [review]
Thought about code docs vs public docs and realised that we do public docs a bit differently. Ignore that 😄.

**Comment by hoebbelsB** [comment]
@BioPhoton @Karnaukhov-kh the build is crashing because only template gets build. It is missing `@rx-angular/cdk` while building though. It relies on that `@rx-angular/cdk` gets build first.
We can fix that either by configuring an `implicitDependency` in `nx.json` or by extending the build command of `@rx-angular/template` with `--with-deps`. 
What is the preferred way of doing so?

**Comment by BioPhoton** [comment]
--with-depth

**Comment by hoebbelsB** [comment]
> --with-depth

this will mean that changes in `@rx-angular/cdk` would make the github actions build it twice. This is not a big deal but it will result in more time consumption for the github actions

**Comment by BioPhoton** [comment]
Hi @hoebbelsB, let me know if there is some progress.

**Comment by hoebbelsB** [comment]
close in favor of #575

**Pull request closed without merging**